index/views.py

```diff
@@ -1,9 +1,31 @@
 """Views module for index app."""
 
 from django.http.request import HttpRequest
+from django.http.response import HttpResponse
 from django.shortcuts import render
 
 
-def index(request: HttpRequest):
-    """Render index page."""
-    return render(request, "base.html")
+def index(request: HttpRequest) -> HttpResponse:
+    """Index view.
+
+    Args:
+        request (HttpRequest): request
```

Comment:
> Типы по формату который был добавлен в Python 3.6 пожалуйста, такой стиль установки типов устарел.

Что конкретно не так? Скобочки?
А то что тут `request: reques` не смущает?

Reply:
> Что конкретно не так? Скобочки?

Типы указываются теперь в коде, а ты указал в док-строке - `(HttpRequest)`

> А то что тут `request: reques` не смущает?

Отвлек и я не заметил. Смущает. Сильно.

Reply:
> Отвлек и я не заметил. Смущает. Сильно.

Ну давай я к каждой вьюшке буду копировать `request: A request that the user makes to get some information by accessing from one of the supported browsers (Chtome-based, Firefox-based, Opera, IE-based) in front of an electronic computing machine (phone/computer/smart vacuum cleaner), while physically existing in the material world on planet Earth. It can be of different types (GET, POST, PUT, FETCH, DELETE), can have different headers and attachments (only in POST request type) and can be initiated by automated software (bots or client from inside javascript code).`

Reply:
[Не так...](https://g.zeos.in/?q=%D0%BA%D0%B0%D0%BA%20%D0%BF%D1%80%D0%B0%D0%B2%D0%B8%D0%BB%D1%8C%D0%BD%D0%BE%20%D0%BF%D0%B8%D1%81%D0%B0%D1%82%D1%8C%20docstring%20%D0%B2%20python)

Reply:
Недостаточно подробно расписал, да?

Reply:
Исправил в https://github.com/fire-square/autodonate/pull/99/commits/c60856b294f4258667a4643a437a51943a432ef8.